apps/api/v2/src/ee/calendars/controllers/calendars.controller.e2e-spec.ts

```diff
@@ -192,9 +192,15 @@ describe("Platform Calendars Endpoints", () => {
       .expect(200);
   });
 
-  it.skip(`/POST/v2/calendars/${OFFICE_365_CALENDAR}/disconnect: it should respond with a 201 returning back the user deleted calendar credentials`, async () => {
+  it(`/POST/v2/calendars/${OFFICE_365_CALENDAR}/disconnect: it should respond with a 201 returning back the user deleted calendar credentials`, async () => {
```

Comment:
please keep the skip here until we add office 365 calendars credentials to test db

Reply:
Ahh yeah my bad, sorry. I can change the tests to make use of the google calendar credentials then.